### PR TITLE
Add login Page

### PR DESCRIPTION
**Pull request by martinMutuma**
###What does this PR do?
Add Login page
Implement Feedback from the Previous PR

### CODE REVIEW

File: src/components/Helpers/Alert.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import { withAlert } from "react-alert";
+
+class ShowAlert extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    if (this.props.message) {
+      this.props.alert.success(this.props.message);
```
Comment (dessHub) [inline src/components/Helpers/Alert.js:13]: Consider destructuring props here.

File: src/components/Login/Login.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+  handleSubmit(event) {
+    event.preventDefault();
+    this.props.display_message("Logging in ........");
+    loginUser(this.state).then(status => {
+      if (status) {
+        this.props.display_message("Login succesfull");
+        this.props.history.push("/");
```
Comment (dessHub) [inline src/components/Login/Login.js:23]: Too many `this.props`, destructure.

**Comment by dessHub** [comment]
Consider ignoring coverage files. Too many files for a single PR

File: src/components/Helpers/Alert.js (hunk 1 above)
Comment (martinMutuma) [reply src/components/Helpers/Alert.js:13]: Done

File: src/components/Login/Login.js (hunk 2 above)
Comment (martinMutuma) [reply src/components/Login/Login.js:23]: Done

**Pull request merged**